docs/docs/sourcing-from-prismic.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Sourcing from Prismic
+---
+
+In this guide, we'll walk through setting up a site with content management using [Prismic](https://prismic.io/).
+
+Prismic is a hosted, proprietary Headless CMS (SaaS) with a web app for creating and publishing content. It's suitable for marketers, editors and developers as it has both a "Writing Room" and a fully-fledged API & Content backend. Besides the usual advantages of a SaaS CMS (hosting, security, updates) Prismic offers features like: Custom type builder, scheduling and content versioning and multi-language support.
+
+Moreover their _Content Slices_ feature enables you to build dynamic layouts by defining reusable custom components and placing them on your landing page, case studies or in your blogposts. Fortunately you can use those in Gatsby to realize features like PrismJS highlighting or using `gatsby-image`. The linked [starter](https://github.com/LekoArts/gatsby-starter-prismic) shows you how to implement that.
+
+The written instructions get accompanied by uncommented YouTube videos which will show the described steps in action. You can also watch the full [video playlist](https://www.youtube.com/playlist?list=PLB-cmN3u7PHJCG-phPyiydhHfiosyd0VC).
+
+_Note: This guide uses the Gatsby Hello World starter to provide a very basic understanding of how Prismic can work with your Gatsby site. If you'd like to start with a full blown template, check out [gatsby-starter-prismic](https://github.com/LekoArts/gatsby-starter-prismic). If you're not familiar with Prismic and its functionalities yet, we highly recommend to check out [Prismic's official documentation](https://prismic.io/docs) which also includes user guides and tutorials. This guide assumes that you have basic knowledge of Prismic & Gatsby (See [Gatsby's official tutorial](/tutorial))._
+
+## Setup
+
+### Prismic
+
+Before initializing the Gatsby project you should sign up for an account on [Prismic.io](https://prismic.io/). The free plan is a perfect fit for personal or smaller projects. Create a new blank repository to get to the content overview of your repository.
+
+Create your first custom type (Repeatable Type) with the name `Post` and add some fields to it. Choose rational names for the `API ID` input while configuring a field because these names will appear in your queries. You should always add the `uid` field in order to have a unique identifier (e.g. for filtering). Then switch to the content overview and create a new document with your `Post` type. Fill out the fields and publish the item.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/yrOYLNiYtBQ?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
Videos help this process a lot, so nice work! I think images are maybe a little easier for some people, but I think video is fine too.